spec/models/generic_file_spec.rb

```diff
@@ -8,6 +8,16 @@
     @file.apply_depositor_metadata(user.user_key)
   end
 
+  it 'is initialized with blanks' do
+    subject.attributes.each do |k,v|
+      if v.is_a?(Array)
```

Comment:
I think you should:

1. Fix this in a new commit
2. Squash all your commits in this branch
2a. (OPTIONAL) if your branch has diverged substantially from master, rebase on the latest in master. I suspect you don't need to do this now, but re: my comment about CLAs, you may need to do that eventually depending on how quickly you can get CLAs signed.
3. Force-push your branch, and this PR will be automatically updated.

How does that sound?